components/widgets/inputs/users_emails_input.tsx

```diff
@@ -179,14 +173,18 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
         );
     }
 
-    onChange = (value: Array<UserProfile | EmailInvite>) => {
+    onChange = (value: ValueType<UserProfile | EmailInvite>) => {
         if (this.props.onChange) {
-            this.props.onChange(value.map((v) => {
-                if ((v as UserProfile).id) {
-                    return v as UserProfile;
-                }
-                return (v as EmailInvite).value;
-            }));
+            if (value) {
```

Comment:
A breaking change in v3 of react-select is that when there are no options (e.g. a multi-select that we have deleted all options) previously it would lead to an empty array, while now it becomes more consistent and becomes a null value (same as initial state).

Reply:
Should we be exposing the webapp's `ReactSelect` package to plugins? I'm wondering what the impact is, on plugins that are currently using v2 of the package

Reply:
That's a good question. Currently we're not exposing it (in the sense that we eg expose luxon or other libraries). Are plugins making assumptions about react-select and its version? cc: @hmhealey probably you have some context around this?

Reply:
I don't think we have any way of guaranteeing that a plugin doesn't depend on a specific version of react-select or any other dependency. Even if we're careful with major version increases, there's still no guarantee that a minor update doesn't break anything.

I'd like to hold off on adding any more shared dependencies until we have plugins using module federation eventually since that has some ability to control the versions of dependencies in a way that's safe. Right now, every package we expose through the window ends up being another spot we may accidentally introduce breaking changes between the web app and plugins